pkg/client/client.go

```diff
@@ -956,15 +969,15 @@ func monitorKeepTerminatedEvent(
 		func(event *chain.KeepTerminatedEvent) {
 			logger.Warningf(
 				"keep [%s] terminated event received at block [%d]",
-				keep.ID(),
+				keep.ID().Hex(),
 				event.BlockNumber,
 			)
 
 			go func(event *chain.KeepTerminatedEvent) {
```

Comment:
The `client.go` file grows and grows... Maybe we should extract the code under this goroutine to a separate file? 
@pdyraga I know you had some thoughts around restructuring the code around client/node. Do you have any ideas on how best we could structure the termination event handling?